src/core/lib/surface/lame_client.cc

```diff
@@ -54,9 +53,9 @@ struct ChannelData {
 static void fill_metadata(grpc_call_element* elem, grpc_metadata_batch* mdb) {
   CallData* calld = static_cast<CallData*>(elem->call_data);
   bool expected = false;
-  if (!calld->filled_metadata.compare_exchange_strong(
-          expected, true, grpc_core::memory_order_relaxed,
-          grpc_core::memory_order_relaxed)) {
+  if (!AtomicCompareExchangeStrong(&calld->filled_metadata, &expected, true,
+                                   std::memory_order_relaxed,
```

Comment:
So, I think @markdroth's comment applies here for consistency about preferring `grpc_core::` to `std::`outside of the `gprpp` directory, though I know that this is likely less important since it's guaranteed to be an enum and not some other sort of class afaik.